pyproject.toml

```
@@ -48,6 +48,52 @@ classifiers = [
     "Programming Language :: Python :: 3.10",
 ]
 
+[project.optional-dependencies]
+tests = [
+    "ansys-dpf-core==0.6.2",
+    "autopep8==2.0.0",
+    "matplotlib==3.6.2",
+    "scipy==1.9.3",
+    "pandas==1.5.1",
+    "pytest==7.2.0",
+    "pytest-cov==4.0.0",
+    "pyvista==0.37.0",
+    "pyansys-tools-report==0.3.2",
+    "vtk==9.0.3",
+    "pytest-rerunfailures==10.2",
+]
+docs = [
```

Could we name this as `doc`? The `pyansys/actions` assume that this is the name for the extras related with documentation. It will be useful when implementing #1632.

```suggestion
doc = [
```